### PR TITLE
Onboarding Page

### PR DESCRIPTION
**Pull request by ainouzgali**

### CODE REVIEW

File: libs/dal/src/repositories/user/user.schema.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ const userSchema = new Schema(
     profilePicture: Schema.Types.String,
     resetToken: Schema.Types.String,
     resetTokenDate: Schema.Types.Date,
+    onBoarding: Schema.Types.Boolean,
```
Comment (scopsy) [inline libs/dal/src/repositories/user/user.schema.ts:14]:
```suggestion
    completedOnBoarding: Schema.Types.Boolean,
```

A small suggestion regarding the schema, maybe worth considering using "completedOnboarding" or shoOnboarding to better give context to what this fields does?

File: apps/web/src/pages/quick-start/QuickStartPage.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    { showOnBoarding: boolean }
+  >(({ showOnBoarding }) => updateUserOnBoarding(showOnBoarding));
+
+  async function disableOnboarding() {
+    await updateOnBoardingStatus({ showOnBoarding: false });
+  }
+
+  async function onDismissOnboarding() {
+    await disableOnboarding();
+    await queryClient.refetchQueries('/v1/users/me');
```
Comment (davidsoderberg) [inline apps/web/src/pages/quick-start/QuickStartPage.tsx:28]: Do we have query key for this or can we create on and use that instead?

File: apps/web/src/pages/quick-start/components/OnboardingSteps.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+import { getActivityStats } from '../../../api/activity';
+
+export const OnboardingSteps = ({ onFinishedAll }: { onFinishedAll: () => void }) => {
+  const navigate = useNavigate();
+
+  const { templates = [] } = useTemplates();
+  const { integrations = [] } = useIntegrations();
+  const { data: activityStats } = useQuery<{
+    yearlySent: number;
+  }>('activityStats', getActivityStats);
```
Comment (davidsoderberg) [inline apps/web/src/pages/quick-start/components/OnboardingSteps.tsx:21]: same here can we use the query key object?

**Review by davidsoderberg — APPROVED** [review]
Looks great, Awesome work 🎉

File: apps/web/src/pages/quick-start/QuickStartPage.tsx (hunk 2 above)
Comment (ainouzgali) [reply apps/web/src/pages/quick-start/QuickStartPage.tsx:28]: I'll add it as a task on linear- if we add that I'd like to do that all around the project, as we use that in a lot of places.

**Pull request merged**